src/pages/projectSettings/tabs/NotificationsTab/SubscriptionField.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from "@emotion/styled";
+import ExpandableCard from "@leafygreen-ui/expandable-card";
+import { Body } from "@leafygreen-ui/typography";
+
+import { Field } from "@rjsf/core";
+import { size } from "constants/tokens";
+import { triggers } from "constants/triggers";
+import { Subscriber } from "gql/generated/types";
+import { ResourceType } from "types/triggers";
+import { toSentenceCase } from "utils/string";
+import { SubscriptionSelect } from "./NotificationsRow";
+
+export const getSubscriberType = (subscriber: Subscriber) => {
+  if (subscriber === undefined) {
+    return "";
+  }
+  let key: string = "";
+  Object.keys(subscriber).forEach((k) => {
+    if (subscriber[k] !== null) {
+      key = k;
+    }
+  });
+  return key;
```

Comment:
Just pointing out that this could also be written with `find`, which may be more readable

Reply:
I don't think I can do that here because subscriber is an object as opposed to an array

Reply:
I think `Object.keys(subscriber).find((k) => subscriber[k] !== null)` would work